omnigibson/scenes/scene_base.py

```diff
@@ -208,7 +205,8 @@ def _load(self):
             )
             og.sim.import_object(self._skybox, register=False)
             self._skybox.color = (1.07, 0.85, 0.61)
-            self._skybox.texture_file_path = m.DEFAULT_SKYBOX_TEXTURE
+            # Default texture to use for skybox
+            self._skybox.texture_file_path = f"{gm.ASSET_PATH}/models/background/sky.jpg"
```

Comment:
why is this no longer a macro?

Reply:
`m.DEFAULT_SKYBOX_TEXTURE` would access gm before users have a chance to change the asset path. Also, this is only used once here, so I just swapped it out with the actual definition.